G/GAP_pkg/GAP_pkg_ferret/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+include("../common.jl")
+
+gap_version = v"400.1190.200"
+name = "ferret"
+version = v"1.0.5"
```

Comment:
The code in this JLL actually depends on the GAP_jll version... Now if I update the GAP_jll in a way that break the ABI, I'll need to recompile this JLL, and the others; so then I should change their versions. So, in conclusion, I guess I should not use "1.0.5" directly but rather transform it similar to what we do for GAP_jll and others... *sigh*